src/BenchmarkDotNet/Templates/WasmAotCsProj.txt

```diff
@@ -37,8 +38,9 @@
     <WasmBuildAppAfterThisTarget>PrepareForWasmBuild</WasmBuildAppAfterThisTarget>
   </PropertyGroup>
   <Target Name="PrepareForWasmBuild" AfterTargets="Publish">
+    <Message Text="!!!debug!!! TargetDir: $(TargetDir)" />
```

Comment:
Do we actually need this `!!!debug!!!` message?

Reply:
Whoops, removed it.